remote-state/dynamostatelock.tf

```diff
@@ -0,0 +1,15 @@
+resource "aws_dynamodb_table" "basic-dynamodb-table" {
```

Comment:
'basic-dynamodb-table' is not the best name. try choosing more meaningful name for this resource

Reply:
changed resouce name